src/OrchardCore.Modules/OrchardCore.OpenId/Settings/OpenIdServerSettings.cs

```diff
@@ -10,15 +13,11 @@ public class OpenIdServerSettings
         public StoreLocation? CertificateStoreLocation { get; set; }
         public StoreName? CertificateStoreName { get; set; }
         public string CertificateThumbprint { get; set; }
-        public bool EnableTokenEndpoint { get; set; }
-        public bool EnableAuthorizationEndpoint { get; set; }
-        public bool EnableLogoutEndpoint { get; set; }
-        public bool EnableUserInfoEndpoint { get; set; }
-        public bool AllowPasswordFlow { get; set; }
-        public bool AllowClientCredentialsFlow { get; set; }
-        public bool AllowAuthorizationCodeFlow { get; set; }
-        public bool AllowRefreshTokenFlow { get; set; }
-        public bool AllowImplicitFlow { get; set; }
+        public PathString AuthorizationEndpointPath { get; set; }
```

Comment:
Before you ask: yes, JSON.NET is deserializes/serializes `PathString` properties flawlessly (because it has a dedicated `TypeConverter`).